package.json

```diff
@@ -71,15 +72,15 @@
     "clang-format": "^1.8.0",
     "commitlint": "^11.0.0",
     "eslint": "^8.19.0",
-    "eslint-config-prettier": "^7.0.0",
+    "eslint-config-prettier": "^8.5.0",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-jest": "^27.4.2",
-    "eslint-plugin-prettier": "^3.1.3",
+    "eslint-plugin-jest": "^26.5.3",
```

Comment:
@zoontek may I ask you why it was needed to downgrade `jest` plugin version?

Reply:
To align with [`@react-native/eslint-config` dependencies version](https://www.npmjs.com/package/@react-native/eslint-config/v/0.74.0?activeTab=code) and avoid installing two different versions (you can check in the `yarn.lock` that it performs a dedup)

Reply:
@zoontek thanks, it makes sense 🙂 👍 (I haven't checked `yarn.lock` diff yet, but I'm in a progress 👀)